src/main/java/com/work/management/service/impl/ProjectService.java

```diff
@@ -0,0 +1,54 @@
+package com.work.management.service.impl;
+
+import com.work.management.entity.Project;
+import com.work.management.repository.EmployeeRepository;
+import com.work.management.repository.ProjectRepository;
+import com.work.management.web.rest.resource.ProjectResource;
+import com.work.management.service.DataPersistingService;
+import com.work.management.utils.ExceptionUtils;
+import lombok.NoArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.Objects;
+
+@Service
+public final class ProjectService implements DataPersistingService<ProjectResource, Project> {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
This should be private static final Logger LOGGER... remember to capitalize all the constants. All static finals are constants.
LoggerFactory.getLogger is a static method...so left side should also be static...and one more thing LoggerFactory.getLogger implements singleton design pattern...there will only be one object of logger per application, that's why also it makes sense to make it static.

Reply:
Just think why Logger needs to implement singleton design pattern.

Reply:
Because it should be thread safe ,if different classes have different logger object then the loggs would get printed in undefined manner